src/components/Person/Person.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+export const PersonLink: React.FC<Props> = ({ person, people }) => {
+  const isWomen = classNames({
+    'has-text-danger': person.sex === 'f',
+  });
+
+  const isMother = people
```

Comment:
The prefix `is` is added for variables that store a boolean value, and here you store a whole object